src/main/java/page_objects/HomePage.java

```diff
@@ -1,4 +1,12 @@
 package page_objects;
 
+import elements.Label;
+import org.openqa.selenium.By;
+
 public class HomePage extends GeneralPage {
+    private final Label lblWelcomeRailWay = new Label(By.cssSelector("#content>h1"));
+
+    public String getWelcomeRailWayText() {
```

Comment:
this is page header or page title, not RailwayText

Reply:
I have fixed